src/bulkExport/startExportJob.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Handler } from 'aws-lambda';
+import AWS from 'aws-sdk';
+import { BulkExportStateMachineGlobalParameters } from './types';
+
+export const startExportJobHandler: Handler<
+    BulkExportStateMachineGlobalParameters,
```

Comment:
Just checking, what params does `Handler` take? Are the two params supposed to be `BulkExportStateMachineGlobalParameters` and `BulkExportStateMachineGlobalParameters` again?

Reply:
Handler is a type that represents a lambda handler function. In this case `BulkExportStateMachineGlobalParameters` is used as both the input event type and the result type.
```js
export type Handler<TEvent = any, TResult = any> = (
    event: TEvent,
    context: Context,
    callback: Callback<TResult>,
) => void | Promise<TResult>;
```